parser/sql/dialect/sqlserver/src/main/antlr4/imports/sqlserver/DMLStatement.g4

```diff
@@ -125,12 +126,16 @@ tableReference
     ;
 
 tableFactor
-    : tableName (AS? alias)? | subquery AS? alias columnNames? | LP_ tableReferences RP_
+    : tableName (AS? alias)?
```

Comment:
Hi @TherChenYang Can you refer this doc for g4 code conduct - https://shardingsphere.apache.org/community/en/involved/conduct/code/#contributor-covenant-g4-of-conduct

> If a rule’s branch is over than 5, every branch take a new line. 

According to g4 code conduct, we should keep these sub rule in one line when the branch is less than 5.